### PR TITLE
Added support for GFS, gfsBlend, and gfsforecast in scenario package

### LINKED ISSUE #1314: `asgs-lint` needs to recognize `BACKGROUNDMET` values of `GFS`, `gfsBlend`
```
(!!! assert_FATAL): 'BACKGROUNDMET' check failed in '/work/06482/x/path/to/asgs_config_XYZ_gfs.sh'!
BACKGROUNDMET - Must be: 'on', 'off', 'NAM', 'OWI', 'namBlend'
 ... fix and rerun linter
```

### PR DESCRIPTION
**Pull request by wwlwpd**
Issue 1314:

Resolves #1314.

### CODE REVIEW

File: bin/asgs-lint [hunk 1]
```diff
@@ -59,17 +59,17 @@ my $assert_checks = {
 #--
 #    TRIGGER must be ftp, rss, rssembedded, or atcf.
 #    NCPUCAPACITY must be greater than or at least equal to NCPU plus NUMWRITERS for each individual scenario.
-#    BACKGROUNDMET set to something other than allowable values : on, NAM, OWI, namBlend, and off.
-#    BACKGROUNDMET parameter can only be set to off or namBlend when TROPICALCYCLONE is set to something other than off.
+#    BACKGROUNDMET set to something other than allowable values : on, NAM, OWI, namBlend,n GFS, gfsBlend, and off.
```
Comment (jasonfleming) [inline bin/asgs-lint:62]: stray `n` typo before GFS

File: bin/asgs-lint [hunk 2]
```diff
@@ -597,12 +597,12 @@ sub fatal_configcheck_SCENARIOPACKAGESIZE_if_TROPICALCYCLONE_DEFINED {
             my ( $found_ENSTORM, $ENSTORM ) = $self->_extract_from_string( q{ENSTORM}, $CASE_block );    # attempt to find value of ENSTORM
             my ( $found_PERCENT, $PERCENT ) = $self->_extract_from_string( q{PERCENT}, $CASE_block );    # attempt to find value of PERCENT
 
-            # check for valid start patterns: 'nhcC', 'veerLeft', 'veerRight', 'namforecast'
+            # check for valid start patterns: 'nhcC', 'veerLeft', 'veerRight', 'namforecast', 'gfsforecast'
```
Comment (jasonfleming) [inline bin/asgs-lint:600]: vMax, rMax, and nhcOfcl are all possibilities as well

**Review by jasonfleming — COMMENTED** [review]
This looks great, there are a couple more common scenario names (prefixes really) that could be added. We don't have a list of "standard" scenario names, they are generally arbitrary. The "veer" etc are used by storm track gen to create the alternative tracks; that is really the only time the ASGS cares what the scenario names are.

File: bin/asgs-lint (hunk 2 above)
Comment (wwlwpd) [reply bin/asgs-lint:600]: nice, I will add them! this is exactly why I needed the review. TY

Comment (wwlwpd) [reply bin/asgs-lint]: @jasonfleming - I updated this in the latest commit

**Review by jasonfleming — APPROVED** [review]
This looks great, thank you! Please merge.

**Pull request merged**